src/v3/src/transformer/oktaVerify/transformAppleSsoExtension.ts

```diff
@@ -58,6 +58,11 @@ export const transformAppleSsoExtension: IdxStepTransformer = ({ formBag, transa
     );
     const isGetMethod = nextStepData?.method?.toLowerCase() === 'get';
 
+    // Apple SSOE fix
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    autoSubmitElement.options.isSSOExtensionVerifyEndpointAlreadyCalled = (nextStepData as any)
```

Comment:
Instead of using a cast to `any`, can we augment the `auth-js` type? Either globally with `*.d.ts` or locally at the top of this file and use a cast to that type:
```ts
type IdxRemediationSSOE extends IdxRemediation {
  isSSOExtensionVerifyEndpointAlreadyCalled?: boolean;
}

// ...

autoSubmitElement.options.isSSOExtensionVerifyEndpointAlreadyCalled = (nextStepData as IdxRemediationSSOE)?.isSSOExtensionVerifyEndpointAlreadyCalled;
```